endpoint.js

```diff
@@ -0,0 +1,15 @@
+const express = require('express');
+const bodyparser =  require('body-parser');
+const mongoose =  require('mongoose');
+
+app.get('/', (req, res) => {
+    res.send('Welcome to APIs!');
+});
+ 
+app.listen(3000, () => console.log('Listening on port 3000'));
```

Comment:
why do we have this?

Reply:
I wanted to use that to test the endpoints